pdata/plog/encoding.go

```diff
@@ -14,6 +14,12 @@
 
 package plog // import "go.opentelemetry.io/collector/pdata/plog"
 
+// MarshalSizer is the interface that groups the basic Marshal and Size methods
+type MarshalSizer interface {
```

Comment:
The convention name is `MarshalerSizer`?

Reply:
Fixing it.